ci/docker/node-runner/Dockerfile.node18

```diff
@@ -0,0 +1,10 @@
+FROM node:18.18.0-alpine3.18@sha256:619ce27eb37c7c0476bd518085bf1ba892e2148fc1ab5dbaff2f20c56e50444d
```

Comment:
This is an image sha for an arm64 image, but these images need to run on concourse which is amd64 and doesn't have any cross-platform capability.

You can get an amd64 SHA by [searching the node image tags](https://hub.docker.com/_/node/tags?page=1&name=18.18.0-alpine) then clicking through to the `linux/amd64` and getting the full sha at the top of that page

Reply:
Agh! Good spot.